datafusion/core/src/datasource/physical_plan/parquet/row_groups.rs

```diff
@@ -632,6 +716,29 @@ mod tests {
             ),
             vec![1]
         );
+
+        // c1 < 5 and c2 IS NULL  => c1_min < 5 and bool_null_count > 0
+        let expr = col("c1")
+            .lt(lit(5))
+            .and(col("c2").eq(lit(ScalarValue::Boolean(None))));
+        let expr = logical2physical(&expr, &schema);
+        let pruning_predicate = PruningPredicate::try_new(expr, schema.clone()).unwrap();
+        let groups = gen_row_group_meta_data_for_pruning_predicate();
+
+        let metrics = parquet_file_metrics();
+        // bool = NULL always evaluates to NULL (and thus will not
```

Comment:
I think this comment is incorrect of date -- both row groups are actually pruned as the vec is empty